conf/cmi/eu_cookie_compliance.cookie_category.essential.yml

```diff
@@ -7,7 +7,7 @@ _core:
kie categories used on the hel.fi site.</td>\r\n\t\t\t<td>365 days</td>\r\n\t\t\t<td>&nbsp;</td>\r\n\t\t</tr>\r\n\t\t<tr>\r\n\t\t\t<td>cookie-agreed</td>\r\n\t\t\t<td>hel.fi</td>\r\n\t\t\t<td>Used by hel.fi Drupal to store information&nbsp;about whether visitors have given or declined the use of cookie categories used on the hel.fi site.</td>\r\n\t\t\t<td>100 days</td>\r\n\t\t\t<td>&nbsp;</td>\r\n\t\t</tr>\r\n\t\t<tr>\r\n\t\t\t<td>cookie-agreed-version</td>\r\n\t\t\t<td>hel.fi</td>\r\n\t\t\t<td>Used by hel.fi Drupal to store information&nbsp;about what version of the cookie consent the user has agreed to.</td>\r\n\t\t\t<td>100 days</td>\r\n\t\t\t<td>&nbsp;</td>\r\n\t\t</tr>\r\n\t\t<tr>\r\n\t\t\t<td>cookie-agreed-categories</td>\r\n\t\t\t<td>hel.fi</td>\r\n\t\t\t<td>Used by hel.fi Drupal to store information&nbsp;about what cookie categories the user has agreed to.</td>\r\n\t\t\t<td>100 days</td>\r\n\t\t\t<td>&nbsp;</td>\r\n\t\t</tr>\r\n\t\t<tr>\r\n\t\t\t<td>AWSELB</td>\r\n\t\t\t<td>siteimproveanalytics.io</td>\r\n\t\t\t<td>\r\n\t\t\t<p>The cookie is related to a load distribution function used to direct requests to servers with the least traffic.</p>\r\n\t\t\t</td>\r\n\t\t\t<td>Session</td>\r\n\t\t\t<td>Third party</td>\r\n\t\t</tr>\r\n\t\t<tr>\r\n\t\t\t<td>AWSELBCORS</td>\r\n\t\t\t<td>siteimproveanalytics.io</td>\r\n\t\t\t<td>\r\n\t\t\t<p>The cookie is related to a load distribution function used to direct requests to servers with the least traffic.</p>\r\n\t\t\t</td>\r\n\t\t\t<td>Session</td>\r\n\t\t\t<td>Third party</td>\r\n\t\t</tr>\r\n\t\t<tr>\r\n\t\t\t<td>mtm_cookie_consent</td>\r\n\t\t\t<td>kartta.hel.fi</td>\r\n\t\t\t<td>\r\n\t\t\t<p>A technical cookie that stores information about how you responded to the notice in the cookie banner about the use of cookies.</p>\r\n\t\t\t</td>\r\n\t\t\t<td>10950 days</td>\r\n\t\t\t<td>&nbsp;</td>\r\n\t\t</tr>\r\n\t\t<tr>\r\n\t\t\t<td>frosmo_quickContext</td>\r\n\t\t\t<td>hel.fi</td>\r\n\t\t\t<td>\r\n\t\t\t<p>The Frosmo cookie improves the user experience and provides targeted content.</p>\r\n\t\t\t</td>\r\n\t\t\t<td>365 days</td>\r\n\t\t\t<td>&nbsp;</td>\r\n\t\t</tr>\r\n\t\t<tr>\r\n\t\t\t<td>id</td>\r\n\t\t\t<td>frosmo.com</td>\r\n\t\t\t<td>\r\n\t\t\t<p>The cookie facilitates visiting the website but does not save information that could be used to identify the visitor.</p>\r\n\t\t\t</td>\r\n\t\t\t<td>5947 days, 15 hours</td>\r\n\t\t\t<td>Third party</td>\r\n\t\t</tr>\r\n\t\t<tr>\r\n\t\t\t<td>JSESSIONID</td>\r\n\t\t\t<td>helsinkikanava.fi</td>\r\n\t\t\t<td>\r\n\t\t\t<p>The cookie is an obligatory cookie that facilitates visiting the website.</p>\r\n\t\t\t</td>\r\n\t\t\t<td>Session</td>\r\n\t\t\t<td>Third party</td>\r\n\t\t</tr>\r\n\t\t<tr>\r\n\t\t\t<td>COOKIE_SUPPORT</td>\r\n\t\t\t<td>helsinkikanava.fi</td>\r\n\t\t\t<td>\r\n\t\t\t<p>The cookie facilitates managing cookies on the website.</p>\r\n\t\t\t</td>\r\n\t\t\t<td>365 days</td>\r\n\t\t\t<td>Third party</td>\r\n\t\t</tr>\r\n\t</tbody>\r\n</table>\r\n"
```

Comment:
Quite a lot of text being removed - is this a change that we want to have?

Reply:
The only thing the above should change is remove the chat related cookies description from the essential group. Those are in their own group. This is also formatted text so it's near impossible to deduce what is going on without checking the before and after in the cookie information page.